src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+@import './mixins';
+
+$content-width: 166px;
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.cards {
+  border: 1px solid #f3f3f3;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  margin: 0 auto;
+  display: flex;
+  box-sizing: border-box;
+
+  &_product-image {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &_title {
+    font-size: 12px;
+    font-weight: 500;
+    margin-top: 40px;
+    width: $content-width;
+    line-height: 18px;
+    margin-bottom: 0;
+    color: #060b35;
+  }
+
+  &_product_code {
+    font-size: 10px;
+    font-weight: 400;
+    color: #616070;
+    text-align: left;
+    width: $content-width;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+}
+
+.cards_stars {
+  width: 100px;
+
+  @include flex-properties;
+
+  &-content {
+    flex-wrap: wrap;
+    justify-content: space-between;
+    width: $content-width;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+  }
+
+  &__star {
```

Comment:
move `stars` block to an own scss file (stars.scss)

Reply:
fixed